### DISCUSSION TITLE
[archived] "Workflow Archive" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#13126] [General]
# en/latest/workflow-archive/

https://argo-workflows.readthedocs.io/en/latest/workflow-archive/

**Comment by PandaHun** [comment]
Hello, Argo experts

we setted archive on our workflows.
record write on mysql successfully, but workflow ui doesn't show nothing about archive

we use rbac in our workflows.
something more need to set on rbac to show archive on ui? 

Additionally, When we delete an archive workflow from ui, there is also appear an window that asks us to delete it from the database.
we wants to disable the db delete function.
Is it possible?

We use argo workflow 3.5.1

**Reply by agilgur5** [reply]
> We use argo workflow 3.5.1

You should really update to the latest patch version, 3.5.7, which has many fixes, including to archived workflows.

Specifically, #13021 and #11715 fixed pagination and load issues with archived workflows in 3.5.

> but workflow ui doesn't show nothing about archive

In 3.5, the Workflows List page shows a merged list of live and archived workflows with a column representing which are archived. Duplicate of #12545 etc al.
It does work, although the merging & pagination may be broken in 3.5.1 as I wrote above.

> something more need to set on rbac to show archive on ui?

No, the directions on this page are complete.
The same RBAC that applies to users viewing live Workflows will apply to them viewing archived workflows.

> When we delete an archive workflow from ui,

This would suggest you do see archived workflows, contradicting your previous statement.

> we wants to disable the db delete function.
> Is it possible?

If you remove `delete` RBAC from your users, they will not be able to delete items from the DB

**Reply by PandaHun** [reply]
Thanks for answer.

But I updated 3.5.7 not appear archived workflows on ui
<img width="58" alt="image" src="https://github.com/argoproj/argo-workflows/assets/13096845/fcb5e236-108b-4b3f-8daf-20fd7a0aeafc">

**Reply by agilgur5** [reply]
As I said previously, "In 3.5, the Workflows List page shows a merged list of live and archived workflows with a column representing which are archived. Duplicate of #12545". There is no separate Archived Workflows page as of 3.5; the two pages were merged together and are shown all in one page as of 3.5.

**Reply by PandaHun** [reply]
oops. Sorry for that.

Thanks your answer.

**Reply by agilgur5** [reply]
I wrote up a PR to note the merged list in the upgrading guide and the new version modal in the UI: https://github.com/argoproj/argo-workflows/pull/13371
